src/utils/createId.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const createId = (data) => {
+  return Math.max(...data.map(element => element.id), 0) + 1;
```

Comment:
Use a more robust method for generating IDs, like UUID, to avoid potential collisions when creating new expenses.
Or make a separate function for find max id & plus 1 to this value